build.gradle

```diff
@@ -16,6 +16,13 @@ if (project.hasProperty('mavenCentral')) {
     apply from: 'gradle/mavenCentral.gradle'
 }
 
+sourceSets {
```

Comment:
is this change necessary?

Reply:
Yes, didn't see the second block when I added this block, so have merged with the `sourceSets` block further down in the file.